test/test_jira.py

```diff
@@ -1,11 +1,15 @@
+import pytest
 from assertpy import assert_that
 
 from label_for_url import determine_label
 
 
-def test_should_label_project() -> None:
+@pytest.mark.parametrize(
```

Comment:
❔ Do you know if we can parameterize all tests in the file in one swoop? :thinking: